charts/telepresence/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+### v2.8.5
```

Comment:
Ah, now I understand why I thought it was old. This entry should be added to the CHANGELOG.md that resides in the repository root, not here. We no longer maintain this one (and we should really remove it to avoid confusion, sorry about that).

Reply:
Fixed, is it good now?
Thanks for your time.